Changelog.md

```diff
@@ -4,6 +4,7 @@ Language Features:
 
 
 Compiler Features:
+* Extracted test cases from SolidityEndToEnd.cpp
```

Comment:
We do not add a changelog entry if the change is not visible to the users.

Reply:
Removed in latest push.